[2022-08-03 15:57:41,672] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:56:28.052191+00:00 [queued]>
[2022-08-03 15:57:41,686] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:56:28.052191+00:00 [queued]>
[2022-08-03 15:57:41,689] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:57:41,692] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-03 15:57:41,695] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:57:41,716] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load-data> on 2022-08-03 15:56:28.052191+00:00
[2022-08-03 15:57:41,731] {standard_task_runner.py:52} INFO - Started process 77 to run task
[2022-08-03 15:57:41,741] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ltv-pipeline', 'load-data', 'manual__2022-08-03T15:56:28.052191+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmp2ez27sw8', '--error-file', '/tmp/tmpkiqzoyap']
[2022-08-03 15:57:41,746] {standard_task_runner.py:80} INFO - Job 55: Subtask load-data
[2022-08-03 15:57:41,866] {task_command.py:371} INFO - Running <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:56:28.052191+00:00 [running]> on host 3b0f05b8acbc
[2022-08-03 15:57:42,151] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ltv-pipeline
AIRFLOW_CTX_TASK_ID=load-data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T15:56:28.052191+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T15:56:28.052191+00:00
[2022-08-03 15:57:42,198] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-08-03 15:57:42,220] {dbapi.py:231} INFO - Running statement: 
            COPY vehicles (id, vin, price, miles, stock_no, year, make, model, trim, body_type, vehicle_type, drivetrain, transmission, fuel_type, engine_size, engine_block, seller_name, street, city, state, zip)
            FROM '/opt/***/files/vehicle_challenge.csv.gz' DELIMITER ',' CSV HEADER;, parameters: None
[2022-08-03 15:57:42,245] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 211, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 235, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedFile: could not open file "/opt/***/files/vehicle_challenge.csv.gz" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.

[2022-08-03 15:57:42,281] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=ltv-pipeline, task_id=load-data, execution_date=20220803T155628, start_date=20220803T155741, end_date=20220803T155742
[2022-08-03 15:57:42,390] {standard_task_runner.py:97} ERROR - Failed to execute job 55 for task load-data (could not open file "/opt/***/files/vehicle_challenge.csv.gz" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.
; 77)
[2022-08-03 15:57:42,494] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-03 15:57:42,930] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
